### PR TITLE
Web server refactor

### PR DESCRIPTION
**Pull request by MattCloward**
Removed sharedCode.js and put it's code in the calculate_score.js
Added and updated comments to a few functions
Removed vcfAttrib from web vcf_parser.js (should make things slightly faster)
Removed extra code

### CODE REVIEW

File: static/js/models/database.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ function handleDisconnect() {
 
     console.log("We ran handleDisconnect...")
     connection.connect(function(err) {
-        console.log("There was an error connection")
+        console.log("There was an error connecting")
```
Comment (mpage21) [inline static/js/models/database.js:22]: we should actually remove this line. It was there for me testing but I forgot to remove it

File: static/js/calculate_score.js [hunk 2]
```diff
@@ -20,12 +20,19 @@ function updateResultBoxAndStoredValue(str) {
     resultJSON = str
 }
 
+/**
+ * Currently unused. Sets the the result box of the Calculate page to str and updates the resultJSON to the new text
```
Comment (mpage21) [inline static/js/calculate_score.js:24]: Why is this currently unused?

File: static/js/calculate_score.js [hunk 3]
```diff
@@ -585,10 +625,25 @@ var ClientCalculateScore = async (snpsInput, associationData, clumpsData, pValue
         }
         var greppedSNPs = new Map();
         greppedSNPs.set("TextInput", greppedSNPsList);
+        return [greppedSNPs, totalInputVariants]
     }
+}
 
+/**
+ * Calculates scores from the file input from the user
```
Comment (mpage21) [inline static/js/calculate_score.js:633]: "... for the file ..." ?

File: static/js/calculate_score.js [hunk 4]
```diff
@@ -630,7 +925,7 @@ function getErrorMessage(err) { //TODO we are going to want to NOT give this inf
 }
 
 /**
- * Returns a simplified output using the given json. The json is truncated and converted to the correct format.
+ * Returns a simplified output using the given JSON. The JSOON is truncated and converted to the correct format.
```
Comment (mpage21) [inline static/js/calculate_score.js:928]: typo

**Review by mpage21 — COMMENTED** [review]
See my comments

File: static/js/calculate_score.js (hunk 2 above)
Comment (MattCloward) [reply static/js/calculate_score.js]: Not sure. It looks like it was useful at some point, but isn't called anywhere. I just deleted it for now.

**Pull request merged**